docs/refguide/api.md

```diff
@@ -15,21 +21,26 @@ Usage:
 * `@observable classProperty = value`
 
 Observable values can be JS primitives, references, plain objects, class instances, arrays and maps.
-`observable(value)` is a convenience overload, that always tries to create the best matching observable types.
+
+**Note:** `observable(value)` is a convenience API that will succeed only if it can be made into
+ an observable data structure (_Array_, _Map_, or _observable-object_). For all other values, no conversion will be performed. 
+
 You can also directly create the desired observable type, see below.
 
-The following conversion rules are applied, but can be fine-tuned by using *modifiers*. See below.
+The following conversion rules are applied, but can be fine-tuned by using [*decorators*](#decorators). See below.
 
 1. If *value* is an instance of an [ES6 Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map): a new [Observable Map](map.md) will be returned. Observable maps are very useful if you don't want to react just to the change of a specific entry, but also to the addition or removal of entries.
 1. If *value* is an array, a new [Observable Array](array.md) will be returned.
 1. If *value* is an object *without* prototype or its prototype is `Object.prototype`, the object will be cloned and all its current properties will be made observable. See [Observable Object](object.md)
-1. If *value* is an object *with* a prototype, a JavaScript primitive or function, a [Boxed Observable](boxed.md) will be returned. MobX will not make objects with a prototype automatically observable; as that is the responsibility of its constructor function. Use `extendObservable` in the constructor, or `@observable` in its class definition instead.
+1. If *value* is an object *with* a prototype, a JavaScript primitive or function, there will be no change made to the value. If you do need a [Boxed Observable](boxed.md), call `observable.box(value)` explicitly. MobX will not make objects with a prototype automatically observable; as that is the responsibility of its constructor function. Use `extendObservable` in the constructor, or `@observable` in its class definition instead.
```

Comment:
... use `@observable` in the class definition, or alternatively `[decorate](link)` or  `[extendObservable](link)` to introduce observable properties on a class definition.